vim_menutrans_helper.py

```diff
@@ -0,0 +1,196 @@
+                    else:
+                        tobe_translated_words.append(word)
+                    if len(word) != 0 and word[-1] == '\\':
+                        pre_backlash = True
+                    else:
+                        pre_backlash = False
+
+                # conver tobe_translated_words list to string
+                for word in tobe_translated_words:
+                    if len(word) > 0:
+                        untranslated_dict[word.lower()] = (
+                            line_number, untranslated_file)
+            elif word_list[0] in toolbar_item:
+                untranslated_dict[word_list[1].lower()] = (
+                    line_number, untranslated_file)
+            elif word_list[0] == "let" and (len(word_list) > 1 and word_list[1].startswith("g:menutrans_")):
+                untranslated_dict[word_list[1].lower()] = (
+                    line_number, untranslated_file)
+
+    # PopUp doesn't need to be translated
+    untranslated_dict.pop("popup", None)
+
+    # blank with digit doesn't neet to be translated
+    temp_untranslated_dict = {}
+    for u in untranslated_dict.keys():
+        if (u.startswith('\\ ') and u.endswith('\\ ')) or (len(u) == 1 and u[0] >= '0' and u[0] <= '9'):
+            temp_untranslated_dict[u] = untranslated_dict[u]
+
+    for key in temp_untranslated_dict.keys():
+        if key in untranslated_dict:
+            del untranslated_dict[key]
+
+
+def makeTranslatedDic(translated_file, translated_dict):
+    translated_menu = {"tmenu", "menut", "menutrans", "menutranslate"}
+    # deal with the translated file
+    with open(translated_file, encoding='utf-8') as f2:
+        for line_number, line in enumerate(f2):
+            # list without '\blank' together per line
+            word_list = line.split()
+            if len(word_list) == 0 or word_list[0][0] == '"':
+                continue
+
+            # list with '\blank' together per line
+            new_word_list = []
+            pre_backlash = False
+            for l in word_list:
+                if pre_backlash:
+                    new_word_list[-1] += " " + l
+                else:
+                    new_word_list.append(l)
+                if l[-1] == '\\':
+                    pre_backlash = True
+                else:
+                    pre_backlash = False
+
+            if new_word_list[0] in translated_menu:
+                translated_dict[new_word_list[1].lower()] = (
+                    line_number, translated_file)
+            if new_word_list[0] == "let" and new_word_list[1].startswith("g:menutrans_"):
+                translated_dict[new_word_list[1].lower()] = (
+                    line_number, translated_file)
+
+
+untranslated_dict = {}
+translated_dict = {}
+makeTranslatedDic("runtime\lang\menu_zh_cn.utf-8.vim", translated_dict)
+
+# traverse runtime dictionary to get the translation difference
+for file in glob.iglob("runtime/**/*.vim", recursive=True):
```

Comment:
Don't write slash directly in path. Instead, always use `os.path.join()` to build up your path.